htdocs/societe/list.php

```diff
@@ -390,23 +404,59 @@
 	}
 } else dol_print_error($db);
 
-$sql = "SELECT s.rowid, s.nom as name, s.name_alias, s.barcode, s.address, s.town, s.zip, s.datec, s.code_client, s.code_fournisseur, s.logo,";
+$sql = "";
+if (!empty($conf->global->MAIN_USE_WITH_INSTEAD_OF_INNER_JOIN)) {
```

Comment:
Such code will not be compatible with a future advanced filter editor.
Why not doing as suggested : No change on sql core request. We keep a simple sql request and the test is just added by added a where ?

Reply:
Ok I modified it